modules/tools/gitlab/main.tf

```diff
@@ -0,0 +1,26 @@
+module "gitlab_namespace" {
+  source = "../../common/namespace"
+
+  resource_limit_cpu = "1"
+  resource_limit_memory = "4G"
+
+  namespace = "gitlab"
+}
+
+resource "helm_release" "gitlab" {
+  count = var.enable_gitlab ? 1 : 0
```

Comment:
Let's move the feature flag from the resource to the parent module. This is a new feature with tf .13.